### DISCUSSION TITLE
Changelog/release notes visibility

### DISCUSSION THREAD

**Discussion by cy6erskunk** [renovatebot/renovate#8501] [Suggest an Idea]
I've just accidentally found that `packageRules.packageNames` is now considered an invalid config option according to the `renovate-config-validator`.
It seems to be working in the GitHub app, so I checked the docs, but there's no `packageNames` field description anymore.
I did some searching to find the [release](https://github.com/renovatebot/renovate/releases/tag/24.30.0) and learned from the https://github.com/renovatebot/renovate/pull/6939 about https://github.com/renovatebot/renovate/issues/6805.
Is there any place where this type of changes could be tracked?
Is there a way to learn which version is now deployed in the GitHub app?
Would it be possible to preseve some information about deprecated config options?

**Comment by rarkins** [comment]
The app contains a migration script to make it invisible there, and we plan to one day raise Pull Requests to migrate the config in each repo too.

Currently the validator is strict and doesn't perform the migration before validating, but we plan to fix/improve that.

In the end, the only time you should need to know about migrated params in future is when you get a PR saying to migrate you config .

**Reply by rarkins** [reply]
This PR should fix it once merged: https://github.com/renovatebot/renovate/pull/8503

**Reply by cy6erskunk** [reply]
This is a great plan!
However, I'm thinking about shared configs in separate repositories: how would it be possible to migrate those?
